### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by IvanTymoshchuk** [IvanTymoshchuk/test-work#2] [Q&A]
Доповни код, присвоївши змінній totalPrice вираз для підрахунку загальної суми замовлення. Змінна pricePerItem зберігає ціну однієї одиниці товару, а orderedQuantity - кількість одиниць товару в замовленні.

**Comment by ViktorSvertoka** [comment]
const pricePerItem = 3500;
const orderedQuantity = 4;

// Change code below this line
const totalPrice = (pricePerItem * orderedQuantity);
